test/src/test/java/org/corfudb/runtime/collections/FGMapTest.java

```diff
@@ -39,26 +39,6 @@ public void canReadWriteToSingle()
                 .isEqualTo("b");
     }
 
-    /*
```

Comment:
I am not sure if FGMaps are used or plan to be supported in future. If we get an agreement lets remove FGMaps too.

Reply:
I thought about that too, but felt that would be too invasive, so chickened out to just removing commented code for now :)

Reply:
Looks like FGMaps are used by CorfuDB-Helloworld repo. If I remove that it will break that example.

Reply:
maybe it's time to remove corfu-helloworld repo? :)

Reply:
Ok, if we are sure that FGMap is not required, I can remove it here, but I do not have the admin privileges for the corfu-helloworld repo. Also should we fix that repo with a better example given FGMap isn't supported?

Reply:
Discuss with Maithem

Reply:
I wouldn't remove it. People can use it as an example of how to build a composable SMR object.